core/modules/wmgr/bin/qservWmgr.py

```diff
@@ -73,13 +72,17 @@ def main():
         app.config.from_pyfile(args.configFile)
     else:
         app.config.from_envvar('WMGRCONFIG')
-    Config.init(app)
+    config.Config.init(app)
 
     # add few blueprints
     app.register_blueprint(dbMgr.dbService, url_prefix='/dbs')
     app.register_blueprint(procMgr.procService, url_prefix='/services')
     app.register_blueprint(xrdMgr.xrdService, url_prefix='/xrootd')
 
+    # check authentication before every request
+    authCheck = auth.Auth(app.config)
+    app.before_request(authCheck.checkAuth)
```

Comment:
Yes, auth is supposed to apply to all served resources, if we need resource-specific we can do it with decorators or looking at the request inside `checkAuth()` method. I actually started with adding decorators everywhere, but it's ugly and there is a chance you can miss a resource or two. 

flask_httpauth decorator is hard to use if you want to switch between basic/digest/hmac in configuration.